tests/test_tftools.py

```diff
@@ -94,3 +95,20 @@ def test_batch_norm_3d():
     x = tf.placeholder(tf.float32, input_shape)
     with pytest.raises(ValueError):
         y = layers.batch_norm(x, '4d')
+
+
+def test_resize_image_like():
+    batch = 1
+    width_height = 5
+    width_height2 = 15
+    channel = 3
+    x = tf.placeholder(tf.float32, [batch, width_height, width_height, channel])
+    s = tf.placeholder(tf.float32, [batch, width_height2, width_height2, channel])
+    x_rshape = images.resize_images_like(x, s)
```

Comment:
reshape rather than rshape. saving one letter is probably not worth readability.
also I prefer reshaped_x - almost reads like a phrase